Doc adding more graphviz commands (#18502)
tldr-pages/tldr: pages/common/gvedit.md

# gvedit

> Simple Graphviz graph editor and viewer.
> More information: <https://graphviz.org/pdf/gvedit.1.pdf>.

- Open a Graphviz file in the editor:

`gvedit {{path/to/graph.gv}}`

> - Scale coordinates from points to inches (useful for layouts generated by Graphviz tools):

Reviewer: ```suggestion
- [s]cale coordinates from points to inches (useful for layouts generated by Graphviz tools):
```

`gvedit -s {{path/to/graph.gv}}`

> - Enable verbose mode:

Reviewer: ```suggestion
- Enable [v]erbose mode:
```

`gvedit -v {{path/to/graph.gv}}`

> - Display usage information:

Reviewer: ```suggestion
- Display help:
```
